### PR TITLE
Fixed problem with strings files

### PR DESCRIPTION
**Pull request by cfva14**
The string files were accepting multiple parameters but no order was specified. 
Fixs  [#1112](https://github.com/commons-app/apps-android-commons/issues/1112)

Example
Before
%s wants to say %s
After
%1$s wants to say %2$s

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,6 @@
+#Tue Jan 30 21:06:06 MST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```
Comment (nicolas-raoul) [inline gradle/wrapper/gradle-wrapper.properties]: Did you intend to commit this file too? If not, would you mind reverting just this file? This can be the topic of another commit if you feel the change is needed. Thanks! :-)

Comment (cfva14) [reply gradle/wrapper/gradle-wrapper.properties]: No. Android Studio updated this one itself. I will fix this. But Let's close this pull request and I will make a new one with all the string files update. deal?

File: app/src/main/res/values-is/strings.xml [hunk 2]
```diff
@@ -210,8 +210,8 @@
   <string name="location_permission_rationale_nearby">Heimild þarf til að birta lista yfir staði í nágrenninu</string>
   <string name="get_directions">FÁ LEIÐSÖGN</string>
   <string name="read_article">LESA GREIN</string>
-  <string name="notifications_welcome">Velkomin á Wikimedia Commons, %s! Við erum ánægð með að þú skulir vera hérna.</string>
-  <string name="notifications_talk_page_message">%s skildi eftir skilaboð á spjallsíðunni þinni</string>
+  <string name="notifications_welcome">Velkomin á Wikimedia Commons, %1$s! Við erum ánægð með að þú skulir vera hérna.</string>
+  <string name="notifications_talk_page_message">%1$s skildi eftir skilaboð á spjallsíðunni þinni</string>
   <string name="notifications_thank_you_edit">Takk fyrir að hafa gert breytingar</string>
-  <string name="notifications_mention">%s minntist á þig á %s.</string>
+  <string name="notifications_mention">%1$s minntist á þig á %2$s.</string>
```
Comment (whym) [inline app/src/main/res/values-is/strings.xml]: One thing I'd be worried about is that in some languages the order might be switched because of a word order different from English. I think this set of changes assumes that all languages have the same word order.

@Nikerabbit If we add' fuzzy="true"' here in the repository, will the bot at TranslateWiki.Net apply the status of those messages into the wiki? If yes, I think that's what we want to do.

Comment (cfva14) [reply app/src/main/res/values-is/strings.xml]: I agree with that. But because I don't speak all this languages, I decided to fix the order of the bot or user notes the words are switched

Comment (Nikerabbit) [reply app/src/main/res/values-is/strings.xml]: I suppose the change in English will cause all translations to be fuzzied even though they are changed too.

**Comment by whym** [comment]
It seemed there remained an unnecessary change in gradle.properties so I removed it https://github.com/cfva14/apps-android-commons/commit/eae795e077b5193758049f609d1aabd0b31064ab
